src/core/Geometry.js

```diff
@@ -119,6 +119,7 @@ export class Geometry {
     }
 
     setInstancedCount(value) {
+        this.isInstanced = value > 0;
```

Comment:
What is the goal in setting the instanced count to 0? The `isInstanced` property is used to determine which gl draw method to use (eg `drawElementsInstanced` or `drawElements`). If 0 instances are required, why would this be useful?

Reply:
By default [instancedCount is set to 0](https://github.com/oframe/ogl/pull/152/files/9fe4b9d0b6d680c15f3a42ee621651497ad84a00#diff-06fce9baac18daa8ba5a200ec1ab78d4f3188d5081e416659a4d9eb3ea11b2b8R40). Maybe someone wants to go back to drawing without instantiating.

Reply:
They could just set that property manually rather than setting the instanced count to `0`.

Reply:
Yes, and they also able to set `this.instancedCount` without `setInstancedCount` method. I did not found no one usage of  `setInstancedCount` method. I decide that this is a public method to manage instancing and `instancedCount` and `isInstanced ` are private fields. May be `setInstancedCount` sould just set `isInstanced` flag to true?

Reply:
In the sense of use-cases, `setInstancedCount` would be an option more frequently used than changing the draw call type (of which I can't really think of of a reason). It's in no way illegal to edit 'private' fields if you're willing. I even go as far as telling users to not be shy of using native gl calls, as this is a lightweight abstraction.

Reply:
Ok, I won't modify the `setInstancedCount` method, but I still don't understand why this method exists.